tests/test_models/test_nn/test_timesfm.py

```diff
@@ -0,0 +1,191 @@
+import os
+from pathlib import Path
+
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from etna.datasets import TSDataset
+from etna.datasets import generate_ar_df
+from etna.libs.timesfm import TimesFmTorch
+from etna.models.nn import TimesFMModel
+from etna.pipeline import Pipeline
+from etna.transforms import DateFlagsTransform
+from etna.transforms import LagTransform
+from etna.transforms import SegmentEncoderTransform
+
+
+@pytest.fixture
+def ts_increasing_integers():
+    n = 128
+    df = generate_ar_df(start_time="2001-01-01", periods=n, n_segments=2)
+    df["target"] = list(range(n)) + list(range(100, 100 + n))
+    ts = TSDataset(df, freq="D")
+    return ts
+
+
+@pytest.fixture
+def expected_ts_increasing_integers():
+    df = generate_ar_df(start_time="2001-05-09", periods=2, n_segments=2)
+    df["target"] = [128.0, 129.0] + [228.0, 229.0]
+    ts = TSDataset(df, freq="D")
+    return ts
+
+
+@pytest.mark.smoke
+def test_chronos_url(tmp_path):
```

Comment:
It isn't chonos, it's timesfm. There are also other places in this file with such mistake.